ui/src/components/overview/TournamanentCard.tsx

```
@@ -190,74 +186,67 @@ export const TournamentCard = ({
+    if (!tournament?.entry_fee.isSome()) {
+      return { type: "free" as const };
     }
-  };
+
+    const normalizedEntryFeeToken = indexAddress(entryFeeToken ?? "");
+    const entryFeeDecimals = tokenDecimals[normalizedEntryFeeToken] || 18;
+    const entryFeePrice = entryFeeToken
+      ? tokenPrices[normalizedEntryFeeToken]
+      : undefined;
+
+    const amount = Number(tournament?.entry_fee.Some?.amount!);
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)